### ISSUE TITLE
add spring security

### ISSUE THREAD

**Issue by reuven9632** [reuven9632/Nail_studio#6]
Provide spring security, the implementation of which will be accompanied by safe url requests such as .../add , .../all , .../{nameOfObject} - to search by name or otherwise, depending on the specifics of the object and the goals of the project, providing  access to various data and site features will be classified from the permissions available to the client, admin, office owner or specialist

**Comment by CodePawfect** [comment]
I guess you want jwt implementation of spring security. If you provide the issuer uri i can implement it.

**Comment by reuven9632** [comment]
yes, I will be glad to your help in the development of my project
https://github.com/reuven9632/Nail_studio/tree/add-spring-security
I was updated this branch
